### PR TITLE
Html tables

### PR DESCRIPTION
**Pull request by KennethDelaCruz**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude! Your indentation here is really wonky, it's hard to read the code. Please clean up your formatting here, and I will take another look.

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <thead>
+          <tr>
+            <th>Student Name</th>
+            <th>Course</th>
+            <th>Grade</th>
+          </tr>
+        </thead>
+        <tbody align="left">
+          <tr>
+            <th>Grace Hopper</th>
```
Comment (codingcodymiller) [inline html-tables/index.html]: Data in the table body should be in `td` elements, not `th` elements.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude! Disregard last review, but this one is for real :joy:. Just a minor change!

File: html-tables/index.html (hunk 1 above)
Comment (KennethDelaCruz) [reply html-tables/index.html:20]: Here are the changes Cody!

image

Comment (KennethDelaCruz) [reply html-tables/index.html]: Here are the changes I made!

image

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+
+  <body>
+    <form>
+      <label for="email">Email</label>
+          <br />
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: Why is this indented?

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <input id="email" type="text" name="Email" size="15"
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: Why is this indented?

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+             max length="30"/>
+             <br />
```
Comment (codingcodymiller) [inline html-forms/index.html:13]: Why is this indented?

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+          <label for="Why is coding cool">Why is Coding Cool?</label>
+              <br />
```
Comment (codingcodymiller) [inline html-forms/index.html:15]: Why is this indented?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
The content of the document here looks good, but the indentation still needs to be revised. Remember, elements that are siblings (in the same parent element) should be at the same level of indentation.

**Comment by thebearingedge** [comment]
Hey @KennethDelaCruz . Please me a new commit to this branch, push again, and open a new PR so we can update your review.

**Pull request merged**